docs/docs/metrics.md

```diff
@@ -4,112 +4,89 @@ title: Metrics
- number of currently processing status updates.
+`service.mesosphere.marathon.core.task.update.impl.ThrottlingTaskStatusUpdateProcessor.processing` (gauge) - The number of status updates currently being processed.
  
  <span class="label label-default">v0.15</span>
- `service.mesosphere.marathon.core.task.update.impl.TaskStatusUpdateProcessorImpl.publishFuture` (timer) -
- times processing status updates.
+ `service.mesosphere.marathon.core.task.update.impl.TaskStatusUpdateProcessorImpl.publishFuture` (timer) - This metric calculates how long it takes Marathon to process status updates.
 
 ### Configuration update processing
 
 <span class="label label-default">v0.15</span>
-`service.mesosphere.marathon.state.GroupManager.queued` (gauge) - number of queued app configuration
-updates. The maximum is configured with `--max_queued_root_group_updates`.
+`service.mesosphere.marathon.state.GroupManager.queued` (gauge) - The number of app configuration updates in the queue. Use `--max_queued_root_group_updates` to configure the maximum.
 
 <span class="label label-default">v0.15</span>
-`service.mesosphere.marathon.state.GroupManager.processing` (gauge) - number of currently processed app configuration
-updates. Since we serialize these updates, this is either 0 or 1.
+`service.mesosphere.marathon.state.GroupManager.processing` (gauge) - The number of currently processed app configuration updates. Since we serialize these updates, this is either 0 or 1.
 
 ### Repositories
 
-Marathon stores its permanent state in "Repositories". These are the most important ones:
+Marathon stores its permanent state in "repositories." The important ones are:
 
 * `GroupRepository` for app configurations and groups.
-* `TaskRepository` for the latest known task state. This is the repository with the largest
-  data churn.
+* `TaskRepository` for the last known task state. This is the repository with the largest data churn.
 
-There are also the
+Other repositories include:
 
 * `AppRepository` for versioned app configuration.
 * `DeploymentRepository` for currently running deployments.
 * `TaskFailureRepository` for the last failure for every application.
-* and more...
 
-For each of the repositories we have statistics about read and write requests. Substitute `*` with
-the repository of interest:
+We have statistics about read and write requests for each repository. To access them, substitute `*` with the name of a repository:
  
-`service.mesosphere.marathon.state.*.read-request-time.count` - the number of read requests.
+`service.mesosphere.marathon.state.*.read-request-time.count` - The number of read requests.
 
-`service.mesosphere.marathon.state.*.read-request-time.mean` - the exponential weighted average of
-the read request times.
+`service.mesosphere.marathon.state.*.read-request-time.mean` - The exponential weighted average of the read request times.
```

Comment:
Perhaps add a link to this: https://dropwizard.github.io/metrics/3.1.0/manual/core/#exponentially-decaying-reservoirs